source/runbooks/set-up-macie.html.md.erb

```diff
@@ -0,0 +1,103 @@
+
+Full details of this creation, including information on the likes of job type, are shown in the code (macie.tf) in the modernisation-platform-environments repo for example and cooker. The appropriate environment account reference should be used as shown below. If this data value isn't in place look on the code to see how it's set up. You may have something more appropriate set up. The comments can be removed but are included here to provide explanation. They are not shown in the example and cooker repositories.
+
+BOTH the sections shown below are needed
+
+NEEDED IN data.tf (or platform_data.tf). In example and cooker they have been included in data.tf
+
+```
+# For macie code
+data "aws_s3_bucket" "bucket1" {
+   bucket = "config-20220407082146408700000002"
+   ...
+   ...
+ }
+ ```
+
+SEPARATE macie.tf code
+
+```
+# Create macie account
+# When logging in as administrator this is not needed but may be for our customers - yet to be fully determined.
+# Account name is MacieAccount. Publishes data every hour (choice are FIFTEEN_MINUTES, ONE_HOUR or SIX_HOURS).
+
+resource "aws_macie2_account" "macieaccount" {
+  finding_publishing_frequency = "ONE_HOUR"
+  status                       = "ENABLED"
+}
+
+# Now create a job
+# Job type options are ONE-TIME and SCHEDULED. If scheduled is chosen you need to select a frequency using schedule_frequency of daily_schedule, weekly_schedule or monthly_schedule
+# The buckets list should be replaced but a list of buckets to examine. This could be limited to those in the account you are connected to or others to which you have access.
+# Below there is a limited list of those from example shown.
+# There are many other job options available and these should be looked at in terrform [job creation](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/macie2_classification_job)
+
+
+resource "aws_macie2_classification_job" "test" {
+  job_type = "ONE_TIME"
+  name     = "<an appropriate job name>"
+  s3_job_definition {
+    bucket_definitions {
+      account_id = local.environment_management.account_ids[terraform.workspace]
+      buckets    = [
+        data.aws_s3_bucket.bucket1.id,
+        data.aws_s3_bucket.bucket2.id,
+        data.aws_s3_bucket.bucket3.id,
+      ]
+    }
+  }
+  depends_on = [ aws_macie2_account.macieaccount ]
+}
+```
+
+Further information can be found in the links below.
+
+Set up a [Macie account](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/macie2_account)
+
+Creating a [Macie job](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/macie2_classification_job)
+
+# How to set up Macie jobs
```

Comment:
This section onwards is console instructions which we don't want users to do so please can it be removed.

Reply:
I thought that might be needed to keep track of creating jobs if users want to.

I did notice we have no details in there of setting up the access to the S3 bucket macie uses and generating the KMS key needed. I suppose this is a "you should know how to that" option for us